src/CMakeLists.txt

```diff
@@ -190,24 +254,33 @@ blt_add_executable(
   stream/TRIAD.cpp
   stream/TRIAD-Seq.cpp
   stream/TRIAD-OMPTarget.cpp
+  stream-kokkos/ADD-Kokkos.cpp
+  stream-kokkos/COPY-Kokkos.cpp
+  stream-kokkos/DOT-Kokkos.cpp
+  stream-kokkos/MUL-Kokkos.cpp
+  stream-kokkos/TRIAD-Kokkos.cpp
+  algorithm/SORT.cpp
+  algorithm/SORT-Seq.cpp
+  algorithm/SORTPAIRS.cpp
+  algorithm/SORTPAIRS-Seq.cpp
+  algorithm-kokkos/SORT-Kokkos.cpp
+  algorithm-kokkos/SORTPAIRS-Kokkos.cpp
   common/DataUtils.cpp
   common/Executor.cpp
   common/KernelBase.cpp
   common/OutputUtils.cpp
   common/RAJAPerfSuite.cpp
   common/RPTypes.hpp
   common/RunParams.cpp
-  algorithm/SORT.cpp
-  algorithm/SORT-Seq.cpp
-  algorithm/SORTPAIRS.cpp
-  algorithm/SORTPAIRS-Seq.cpp
   DEPENDS_ON ${RAJA_PERFSUITE_DEPENDS}
 )
 
 else()
+	if(NOT INFRASTRUCTURE_ONLY)
```

Comment:
This looks unnecessary.

Reply:
Hi David -- Please see my comment above on this point (i.e., there are two distinct scenarios in which we're using RAJAPerf Suite).  Would it be possible for us to meet, and work through at least some / most of your suggested changes?  I just want to make sure I'm doing what you want me to do for the merge.